mobile-app/app/screens/AppNavigator/screens/Loans/components/AddOrRemoveCollateralForm.tsx

```diff
@@ -164,6 +206,16 @@ export const AddOrRemoveCollateralForm = React.memo(({ route }: Props): JSX.Elem
         suffix={` ${token.displaySymbol}`}
         styleProps={tailwind('font-medium')}
       />
+      <View style={tailwind('flex flex-row justify-between')}>
+        <ThemedText>{translate('components/AddOrRemoveCollateralForm', 'Resulting collateralization')}</ThemedText>
+        <ThemedText
+          style={tailwind('font-semibold')}
+          light={colors.light}
+          dark={colors.dark}
+        >{resultingColRatio.isNegative() || resultingColRatio.isNaN() ? translate('components/AddOrRemoveCollateralForm', 'N/A') : `${resultingColRatio.toFixed(2)}%`}
```

Comment:
need to handle content overflow
![IMG_05E5DFD2F2AD-1](https://user-images.githubusercontent.com/40191153/146319006-bad3dd58-fff6-4e31-aec1-4d1315898372.jpeg)

Reply:
let's display N/A when user attempt to remove all of their collaterals
![IMG_13C5DCD4E936-1](https://user-images.githubusercontent.com/40191153/146319307-6a034905-3df2-4e49-8037-8c68c43c61bb.jpeg)

Reply:
Done with these 2
1. Added a horizontal`scrollview` to handle long values for `resulting collateralization`
2. `N/A` will display whenever the resulting collateralization is <=0

Reply:
1. can we wrap it to new line? can ignore if you have consulted jonathan
2. ideally the `N/A` should not be color-coded.

Reply:
1. Will consult with @jtlntno-cake on this one. I think it's releasable in its current state. I'll iterate on the next release depending on Jon's comment
2. Done. Updated!